frontend/src/App.tsx

```diff
@@ -22,7 +22,12 @@ import Homepage from './components/pages/misc/homepage.component';
 import About from './components/pages/misc/about.component';
 import Profile from './components/pages/user/profile.component';
 
-import { IAppState, IUser } from 'store/reduxState';
+import {
+  createAuthenticatedAction,
+  createNotAuthenticatedAction,
+  IAppState,
+  IUser,
+} from 'store/reduxState';
```

Comment:
These "create" actions provide a better more consistent way to build actions to dispatch to the redux store.